jobs/build/ocp3/Jenkinsfile

```diff
@@ -732,13 +732,13 @@ node {
 
 
             // Push the building puddle out to the correct directory on the mirrors (e.g. online-int, online-stg, or enterprise-X.Y)
-            buildlib.invoke_on_rcm_guest("3.11-plashet-to-rcm-guest.sh", SYMLINK_NAME, NEW_FULL_VERSION, BUILD_MODE)
+            buildlib.invoke_on_ocp_artifacts("3.11-plashet-to-ocp-artifacts.sh", SYMLINK_NAME, NEW_FULL_VERSION, BUILD_MODE)
```

Comment:
As mentioned above, this might not be needed anymore.

Reply:
fixed in e8db8cea361bb82124bee6bf6db92f44c1e7736e